src/server/server.ts

```diff
@@ -121,6 +133,11 @@ export class CTProtoServer<AuthRequestPayload, AuthData, ApiRequest extends NewM
       let msgWaiter: NodeJS.Timeout;
       const msgWaitingTime = 3000;
 
+      /**
+       * Set type of incoming binary data to Buffer, because client uses buffer type of binary data
```

Comment:
Sorry, I still can not understand what it means. Maybe you can explain it in more detail?